flow/connectors/postgres/postgres.go

```diff
@@ -782,9 +782,16 @@ func (c *PostgresConnector) SetupReplication(signal *SlotSignal, req *protos.Set
 		return fmt.Errorf("error checking for replication slot and publication: %w", err)
 	}
 
+	tableNameMapping := make(map[string]model.NameAndExclude)
+	for k, v := range req.TableNameMapping {
```

Comment:
req is `protos.SetupReplicationInput`. Could pull exclusions & then populate a new TableNameMapping field to carry exclusions through. But since it works, not sure this is necessary. Maybe useful to somehow stop source from sending excluded columns altogether?

Reply:
this seems good for now :-)